### DISCUSSION TITLE
Live Update: partial bundle

### DISCUSSION THREAD

**Discussion by mustafa0x** [capawesome-team/capacitor-plugins#245] [Ideas]
The plugin requires the new bundle to be the entire `webDir` folder. I have several large files in `webDir` that don't change much. Including them in each live update bundle increases size, which increases the risk of updates failing to download or apply.

Could I mark a bundle as "partial", so that instead of the bundle overwriting the entire `webDir` folder, its contents are merged with `webDir`?

**Comment by robingenz** [comment]
No, this feature is not yet available but it's planned. I close this as duplicate of https://github.com/capawesome-team/capacitor-plugins/issues/180.

**Reply by mustafa0x** [reply]
Thanks for the quick reply. I did search, but didn't see that issue.

Until we get that feature, does capacitor support placing assets in directory other than `webDir`? If that exists, we can put rarely-changing-assets in it instead.

**Reply by robingenz** [reply]
Capacitor itself does not. But Android and iOS builds can of course contain static assets, just as the `webDir` folder is also a static asset. Otherwise, you could download these files once at startup. This is what the [ML Kit SDK](https://developers.google.com/ml-kit) does for large ML models, for example.

Anyway, the issue has been prioritized and I plan to start development soon.

**Reply by mustafa0x** [reply]
Thanks, useful info, I truly appreciate your response and time!

**Reply by robingenz** [reply]
You're welcome!
